src/Paramore.Brighter.MessagingGateway.AWSSQS/SqsMessageProducer.cs

```diff
@@ -60,7 +60,7 @@ public bool ConfirmTopicExists(string topic = null)
                    _publication.MakeChannels);
            }
 
-           return string.IsNullOrEmpty(ChannelTopicArn);
+           return !string.IsNullOrEmpty(ChannelTopicArn);
```

Comment:
@preardon  Can you review this please as it will effect you maybe?

Reply:
This is good, looks like a bug on the return value